### ISSUE TITLE
Prevent showing tooltip on popover

### ISSUE THREAD

**Issue by ivictbor** [twbs/bootstrap#16030]
Hi! I need < a > tag which shoud display tooltip on cursor moving and should drop popover on clicking. But tooltip is also shown when I move cursor above popover. Check it: https://jsfiddle.net/ivictbor/re15cuby/

**Comment by twbs-lmvtfy** [comment]
Hi @ivictbor!

You appear to have posted a live example (https://fiddle.jshell.net/ivictbor/re15cuby/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 15, column 7: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.2 ; saw what appears to be usage of Bootstrap 3.0.0
* line 19, column 7: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.2 ; saw what appears to be usage of Bootstrap 3.0.0

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by ivictbor** [comment]
linked 3.3.2 https://jsfiddle.net/ivictbor/re15cuby/2/ , issue still exists

**Comment by kkirsche** [comment]
I'm not sure I understand but that could be cause I'm on a mobile device at the moment. But for me everything is displaying as I would expect unless there is something I'm missing.

**Comment by cvrebert** [comment]
Please explain what you mean by "on cursor moving". Do you mean hover?

**Comment by cvrebert** [comment]
Duplicate of http://stackoverflow.com/questions/28951100/prevent-showing-tooltip-on-popover

**Comment by cvrebert** [comment]
Based on the SO conversation, this isn't a bug *per se*, but rather an artifact of how the "sibling" tooltip/popover insertion strategy works, which can be avoided by setting the `container: 'body'` option and thus choosing the alternative "body" insertion strategy.
It's definitely not great, but it's not really fixable without rewriting the tooltip plugin.
We're looking at ways to improve things in v4 though.

**Comment by ivictbor** [comment]
Many thanks cvrebert ! Works great!
